Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -343,25 +343,20 @@ public void UnregisterPointer(IPointingSource pointingSource)
 
         public GameObject TryGetFocusedObject(BaseEventData eventData)
         {
-            FocusDetails? details = TryGetFocusDetails(eventData);
-
-            if (details == null)
-            {
-                return null;
-            }
-
             IPointingSource pointingSource;
             TryGetPointingSource(eventData, out pointingSource);
             PointerInputEventData pointerInputEventData = GetSpecificPointerEventData(pointingSource);
 
             Debug.Assert(pointerInputEventData != null);
-            pointerInputEventData.selectedObject = details.Value.Object;
-
-            return details.Value.Object;
+            return pointerInputEventData.selectedObject;
         }
 
         public bool TryGetPointingSource(BaseEventData eventData, out IPointingSource pointingSource)
         {
+            // pre-initialize pointingSource to null, assuming we will return false
+            pointingSource = null;
+            if (eventData == null) { return false; }
```

Comment:
Not sure how/why the event data could be null. 

Seems like it's just treating the symptom not the cause.

Reply:
I agree, this is just hiding the problem rather than fixing the issue.
I guess we'll review in vNext